spec.md

```diff
@@ -493,6 +493,18 @@ message CreateVolumeRequest {
   // sensitive and MUST be treated as such (not logged, etc.) by the CO.
   // This field is OPTIONAL.
   map<string, string> controller_create_credentials = 6;
+
+  // The topological segment(s) the provisioned volume MUST be
+  // accessible from.
+  // A topological segment is a sub-division of a cluster, like region,
+  // zone, rack, etc.
+  // An SP SHALL advertise the requirements for topology in
+  // documentation. COs SHALL permit passing through topological
+  // information.
+  // This field is OPTIONAL. If it is not specified, the SP MAY choose
+  // the topological segment(s) the provisioned volume is accessible
+  // from.
+  TopologyRequirement accessibility_requirements = 7;
```

Comment:
naming: I would prefer "TopologyPolicy accessible_from"

Reply:
TODO: include this field in the compatibility check described in the paragraph (above this message type) that begins "If a volume corresponding to the specified volume `name` already exists..."

Reply:
Agreed `accessible_from` I like more than `accessibility_requirements`, will change.
And will do on adding to compatibility check.